packages/flutter/lib/src/widgets/two_dimensional_viewport.dart

```diff
@@ -0,0 +1,785 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'framework.dart';
+import 'scroll_notification.dart';
+
+export 'package:flutter/rendering.dart' show AxisDirection, GrowthDirection;
+
+/// A delegate that supplies children for scrolling in two dimensions.
+///
+/// A [TwoDimensionalScrollView] lazily constructs its box children to avoid
+/// creating more children than are visible through the
+/// [TwoDimensionalViewport]. Rather than receiving children as an
+/// explicit [List], it receives its children using a
+/// [TwoDimensionalChildDelegate].
+abstract class TwoDimensionalChildDelegate extends ChangeNotifier {
+  /// Creates a delegate that supplies children for a [TwoDimensionalScrollView].
+  TwoDimensionalChildDelegate({ this.mainAxis = Axis.vertical});
+
+  /// Designates the primary axis of the [TwoDimensionalScrollView].
+  ///
+  /// For the given [Axis], the nearest [PrimaryScrollController] surrounding
+  /// the widget is attached to the scroll position of that Axis.
+  ///
+  /// Defaults to [Axis.vertical].
+  final Axis mainAxis;
+
+  /// Returns the child with the given indices.
+  ///
+  /// Should return null if asked to build a widget with a greater
+  /// index than exists. If this returns null, [estimatedChildCount]
```

Comment:
Does `estimatedChildCount` still exist? I couldn't find it on this change.

Reply:
Oh! That is a bit of copy pasta. Thank you!